mmdet3d/ops/furthest_point_sample/src/furthest_point_sample.cpp

```diff
@@ -19,6 +19,16 @@ void furthest_point_sampling_kernel_launcher(int b, int n, int m,
                                              const float *dataset, float *temp,
                                              int *idxs, cudaStream_t stream);
 
+int furthest_point_sampling_with_dist_wrapper(int b, int n, int m,
```

Comment:
As we borrow the code from another repo, we should clarify at the top of this file.

Reply:
Also, add the link of the original code in python functions.

Reply:
> As we borrow the code from another repo, we should clarify at the top of this file.

Add link in the cuda function should be OK.